Cubical/Codata/M/AsLimit/M/Base.agda

```diff
@@ -121,7 +121,7 @@ shift-iso S@(A , B) =
         PathP (λ x → B (p n x) → Wₙ S n) (πₙ S ∘ u (suc n)) (u n)
           Iso⟨ pathToIso (PathP≡Path (λ x → B (p n x) → Wₙ S n) (πₙ S ∘ u (suc n)) (u n)) ⟩
         subst (λ k → B k → Wₙ S n) (p n) (πₙ S ∘ u (suc n)) ≡ (u n)
-          Iso⟨ (invIso (temp (pathToIso (cong (λ k → B k → Wₙ S n) (α-iso-step-5-Iso-helper0 a p n))))) ⟩
+          Iso⟨ (invIso (temp' (pathToIso (cong (λ k → B k → Wₙ S n) (α-iso-step-5-Iso-helper0 a p n))))) ⟩
```

Comment:
Instead of using pathToIso here, wouldn't it just be easier to prove that cong applied to an iso is an iso? It's probably very expensive to use pathToIso everywhere

Reply:
Can't you just use https://github.com/agda/cubical/blob/master/Cubical/Foundations/Equiv/HalfAdjoint.agda#L116 ?

Reply:
I will try and have a look at it. Seems like it should be usable.

Reply:
It seems like it does the same as `iso→fun-Injection-Iso-x` but where the input is an equivalence instead of an isomorphism, so I don't think it helps.